docker/generic/start_proxy.py

```diff
@@ -955,7 +955,7 @@ def gen_envoy_args(args):
            "--log-format-escaped"]
 
     if args.enable_debug:
-        cmd.append("-l debug")
+        cmd.append("-l debug --component-log-level upstream:info,main:info")
```

Comment:
Can we just exclude DNS,  I think upstream and main have other useful debug info?  e.g. main has envoy config info.
upstream connection info is important too.

Reply:
Unfortunately DNS are debug logs in the `upstream` component, stats are debug logs in the `main` component: https://screenshot.googleplex.com/85850ba9-6083-4247-be95-3947fd715cde

I did some analysis, this shows the other logs that we'd lose with this PR: https://screenshot.googleplex.com/AqbNqAavU5gmkQd

Do you think this are useful to have? I don't see the value these ones add.

If you think we need those logs as well, then I'll revert this PR. Seems there is no other way to filter out the noise other than by component.

Reply:
OK, then